complaints/templates/complaints/complaints_register.html

```diff
@@ -1,11 +1,104 @@
-</form>
+	{% if user.is_authenticated %} 
+	{% if user.is_staff or user.is_nucleus %}
+	  <li><a href="{% url 'previous-requests' %}">All Complaints</a></li>
+	{%else%}
+	  <li><a href="{% url 'previous-requests' %}">My Complaints</a></li>
+	{%endif%}
+	{% if not user.is_staff and not user.is_nucleus %}
+      <li class="active"><a href="{% url 'complaint-register' %}">Register Complaints</a></li>
+	{%endif%}
+	<li><a href="{% url 'unblock-request' %}">URL Unblock Request</a></li>
+	{%endif%}
+      <li><a href="/contact">Contact Us</a></li>
+    </ul>
+    <ul class="nav navbar-nav navbar-right">
+	{% if user.is_authenticated %} 
+      <li><a href="/logout"><i class="fa fa-power-off"></i> Logout</a></li>
+	{%else%}
+	  <li><a href="/login"><span class="glyphicon glyphicon-user"></span> Login</a></li>
+	{%endif%}
+    </ul>
+    </div>
+  </div>
+</nav>
+      <div class="container">
+         <div class="row">
+            <div class="col-md-6 col-md-offset-3">
+               <form class="form-horizontal" action="{% url 'complaint-register' %}" method="post">
+                  {% csrf_token %}
+                  <fieldset>
+                     <legend class="text-center">Complaint Details</legend>
+                     <div class="form-group">
+                        <center>
+                           {{ message }}
+                           {% if message|length %}<br><br>{%endif%}
+                           <select id="option" name="category" class="form-control form-control-sm" a="true" style="width:200px;">
+                              <option value="none">Choose a Category</option>
+                              <option value="wifi">WiFi</option>
+                              <option value="lan">Lan</option>
+                              <option value="firewall">Sophos Firewall</option>
+                           </select>
+                        </center>
+                     </div>
+                     <div id="other3" style="display:none;">
+                        <div class="form-group">
+                           <label class="col-md-3 control-label" for="message" >Complaint Details</label>
+                           <div class="col-md-9">
+                              <textarea class="form-control" a="true" id="message" name="remark" placeholder="Please enter your message here..." rows="5"></textarea>
+                           </div>
+                        </div>
+                        <div class="form-group row">
+                           <label class="col-md-3 control-label" for="message">-</label>
+                           <div class="col-md-9">
+                              <label for="default" class="btn btn-default" onclick="urg_fun()" style="position:relative;">
+                                 <b>Urgency</b><input type="checkbox" id="default" class="badgebox"><!-- render_field form.urgency id="default" class="badgebox" %}--><span class="badge">&check;</span>
```

Comment:
useless comment and isn't there an attribute on the checkbox input to mark / unmark it?